include/grpc/support/metrics.h

```diff
@@ -28,7 +28,8 @@ class StatsPluginChannelScope {
  public:
   StatsPluginChannelScope(absl::string_view target,
                           absl::string_view default_authority)
-      : target_(target), default_authority_(default_authority) {}
+      : target_(std::string(target)),
```

Comment:
Nit: No need to explictly say `std::string()` on these two lines, since we're already passing the parameters to `std::string` ctor by virtue of the data members now being that type.

Reply:
Thanks!